Source/MediaInfo/File__Analyze_Streams_Finish.cpp

```diff
@@ -1137,6 +1137,10 @@ void File__Analyze::Streams_Finish_StreamOnly_General(size_t StreamPos)
         if (!CompanyName.empty() && Model.rfind(CompanyName + __T(' '), 0) == 0) {
             Fill(Stream_General, StreamPos, General_Encoded_Hardware_Model, Model.substr(CompanyName.size() + 1), true);
         }
+        if (CompanyName == __T("NIKON CORPORATION") && Model.rfind(__T("NIKON "), 0) == 0) {
+            Fill(Stream_General, StreamPos, General_Encoded_Hardware_CompanyName, Ztring("Nikon"), true);

```

Comment:
```suggestion
            Fill(Stream_General, StreamPos, General_Encoded_Hardware_CompanyName, "Nikon", true, true);
```

Better to use char* version when possible (and it will be easier for us when we switch to UTF-8 in our codebase).

Reply:
Need to include the size else it won't build due to multiple matching overloads.